styles/components/_type.scss

```diff
@@ -41,6 +41,10 @@ h6, .h6, .tx6, .btn.tx6 {
   font-size: $txBg;
 }
 
+.txTiny {
```

Comment:
Can we stick with the current naming convention for Tiny and make this txTn?

In other places we're using the convention Tn, Sm, Md, Lg, Hg. 

A few places have Bg for Big instead of Lg for Large, that'll have to be cleaned up.